### PR TITLE
Add new generated classes and json test

### PR DESCRIPTION
**Pull request by jackhopner**

### CODE REVIEW

File: .apidoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 code:
-  movio:
+  movio.cinema:
     apidoc-generator-reference:
-      version: 0.1.0-SNAPSHOT 
+      version: 0.0.1-SNAPSHOT
```
Comment (jerrypnz) [inline .apidoc:4]: `0.1.0` was changed to `0.0.1`? That's...interesting ;-)

File: lib/src/test/scala/JsonParsingTest.scala [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  import movio.cinema.apidoc.generator.reference.v0.models.json._
+
+  describe("datetime validation") {
+    it("should throw an jsresultexception if the datetime has no tz offset") {
+      intercept[JsResultException] {
+        val json = """{"id":"1","name":"John","lastActiveTime":"2016-12-24T15:36:54","addresses":[]}"""
+
+        Json.parse(json).as[Person]
+      }
+    }
```
Comment (jerrypnz) [inline lib/src/test/scala/JsonParsingTest.scala:31]: Add empty lines between test cases?

File: lib/src/test/scala/JsonParsingTest.scala [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    it("should UTC as tz offset") {
+      val json = """{"id":"1","name":"John","lastActiveTime":"2016-12-24T15:36:54Z","addresses":[]}"""
+
+      Json.parse(json).as[Person]
```
Comment (jerrypnz) [inline lib/src/test/scala/JsonParsingTest.scala]: Add `shouldBe` and verify that the date time is the same as expected?

File: lib/src/test/scala/JsonParsingTest.scala [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    it("should support tz offsets") {
+      val json = """{"id":"1","name":"John","lastActiveTime":"2016-12-24T15:36:54+0400","addresses":[]}"""
+
+      Json.parse(json).as[Person]
```
Comment (jerrypnz) [inline lib/src/test/scala/JsonParsingTest.scala]: Same here

File: lib/src/test/scala/JsonParsingTest.scala [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    }
+    it("should support milliseconds") {
+      val json = """{"id":"1","name":"John","lastActiveTime":"2016-12-24T15:36:54.932+0400","addresses":[]}"""
+
+      Json.parse(json).as[Person]
```
Comment (jerrypnz) [inline lib/src/test/scala/JsonParsingTest.scala]: Same here

File: .apidoc (hunk 1 above)
Comment (jerrypnz) [inline .apidoc:4]: Ah because the organization was changed to `movio.cinema`?

Comment (jackhopner) [inline .apidoc:4]: yep

File: lib/src/test/scala/JsonParsingTest.scala (hunk 2 above)
Comment (jackhopner) [inline lib/src/test/scala/JsonParsingTest.scala:31]: done

File: lib/src/test/scala/JsonParsingTest.scala (hunk 3 above)
Comment (jackhopner) [inline lib/src/test/scala/JsonParsingTest.scala]: done

File: lib/src/test/scala/JsonParsingTest.scala (hunk 4 above)
Comment (jackhopner) [inline lib/src/test/scala/JsonParsingTest.scala]: done

File: lib/src/test/scala/JsonParsingTest.scala (hunk 5 above)
Comment (jackhopner) [inline lib/src/test/scala/JsonParsingTest.scala]: done

**Pull request merged**